fineract-core/src/main/java/org/apache/fineract/infrastructure/core/service/database/JdbcJavaType.java

```diff
@@ -0,0 +1,321 @@
+package org.apache.fineract.infrastructure.core.service.database;
+
+import jakarta.validation.constraints.NotNull;
+import java.sql.JDBCType;
+import org.apache.fineract.infrastructure.core.exception.PlatformServiceUnavailableException;
+
+public enum JdbcJavaType {
```

Comment:
Same as for JavaType.

Reply:
Same answer, thanks

Reply:
See previous comment for JavaType, looks like a job for JOOQ.